screens/Settings/DataBackup.tsx

```diff
@@ -0,0 +1,62 @@
+import React, {useState} from 'react';
+import {Pressable} from 'react-native';
+import {Icon, ListItem} from 'react-native-elements';
+import {Text} from '../../components/ui';
+import {Theme} from '../../components/ui/styleUtils';
+import {useBackupScreen} from './BackupController';
+import {BackupToggle} from './BackupToggle';
+
+export const DataBackup: React.FC = ({} = props => {
+  const controller = useBackupScreen(props);
+
+  // TODO : Check if the setup is already done
```

Comment:
What is this ToDo about?

Reply:
This to-do was added when we set up the password and phone number for cloud backup. As this story is for local backup creation, This TODO can be removed.